python/cuml/test/test_random_forest.py

```diff
@@ -721,4 +720,4 @@ def test_rf_classification_proba(datatype, split_algo, rows_sample, nrows,
         sk_model.fit(X_train, y_train)
         sk_preds_proba = sk_model.predict_proba(X_test)
         sk_mse = mean_squared_error(y_proba, sk_preds_proba)
-        assert fil_mse <= (sk_mse+0.004)
+        assert fil_mse <= (sk_mse + 0.0061)
```

Comment:
Per discussion, I suggest you add a comment with the initial MSEs observed from each. That will make the magnitude clear and also help tracking any future issues.